.reuse/dep5

```diff
@@ -0,0 +1,40 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Nextcloud Vue components library
+Upstream-Contact: Nextcloud GmbH <https://nextcloud.com/impressum/>
+Source: https://github.com/nextcloud-libraries/nextcloud-vue
+
+Files: package.json package-lock.json .github/pull_request_template.md
+Copyright: Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+Files: styleguide/assets/default.css styleguide/assets/icons.css styleguide/assets/server.css cypress/snapshots/* tests/unit/functions/usernameToColor/__snapshots__/usernameToColor.spec.js.snap
+Copyright: Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+Files: l10n/*.pot
+Copyright: 2020-2024 Nextcloud translators
+License: AGPL-3.0-or-later
+
+Files: tsconfig.json cypress/tsconfig.json tests/tsconfig.json
+Copyright: 2022-2024 Nextcloud GmbH and Nextcloud contributors
+License: CC0-1.0
+
+Files: resources/timezones/zones.json
+Copyright: Internet Assigned Numbers Authority
+License: CC0-1.0
+
+Files: styleguide/assets/img/favicon*.* styleguide/assets/img/logo/logo*.*
+Copyright: 2016-2024 Nextcloud GmbH
+License: LicenseRef-NextcloudTrademarks
+
+Files: styleguide/assets/close.svg styleguide/assets/img/filetypes/*
+Copyright: 2020 Google Inc.
+License: Apache-2.0
```

Comment:
Are all the filetypes images Google images?
Also, could you add extensions here to the glob to have less risk of having wrong files?

And what about other images?

Reply:
All mdi now

Reply:
Resolved